kernel/userspace.c

```diff
@@ -255,26 +255,26 @@ static void thread_idx_free(uintptr_t tidx)
 
 struct z_object *z_dynamic_object_create(size_t size)
 {
-	struct dyn_obj *dyn_obj;
+	struct dyn_obj *obj_thread;
```

Comment:
`obj_thread` not a good name, it isn't a thread.

call it `dyn` maybe? it's just a local variable